packages/rollup/src/rollup_bundle.bzl

```diff
@@ -98,7 +99,7 @@ rollup_bundle(
 )
 ```
 """,
-        allow_single_file = [".js"],
+        allow_single_file = True,
```

Comment:
can it be constrained at all?

it's easier to make this looser over time but never tighter

Reply:
rollup bundle can take pretty much anything as an input given that a user can add any plugins so I think the only way to support everything is to be unconstrained